scripts/labman

```diff
@@ -69,6 +71,8 @@ def config():
     click.echo('Main configuration:')
     test_env = click.prompt('Test environment', default=True)
     log_dir = click.prompt('Log directory', default="/tmp/")
+    certificate_file = click.prompt('Certificate file', default="")
```

Comment:
Since there is another prompt later in the config set-up that asks for the "Qiita server certificate", I would like to see this prompt be something distinct but parallel, like "Labman server certificate" (or change the Qiita one to parallel this one as "Qiita certificate file").